File: web-app/src/model/Car.js


import React, { Component } from 'react'

export class Car {
    constructor(data) {
        if (!data)
            return;
        this.id = data.id
        this.make = data.carMake
        this.model = data.carModel
        this.licenseNumber = data.licenseNumber
        this.seats = data.seats
        this.year = data.year
        this.location = data.location
        this.price = data.price
    }
}

export default Car

File: Backend/src/main/java/com/tazi34/carservice/car/Car_.java
package com.tazi34.carservice.car;


import com.tazi34.carservice.carlocation.spot.Spot;

import javax.persistence.metamodel.SingularAttribute;
import javax.persistence.metamodel.StaticMetamodel;
import java.math.BigDecimal;

@StaticMetamodel(Car.class)
public class Car_ {
    public static volatile SingularAttribute<Car,Long> id;
    public static volatile SingularAttribute<Car, String> model;
    public static volatile SingularAttribute<Car, String> make;
    public static volatile SingularAttribute<Car, Integer> seats;
    public static volatile SingularAttribute<Car,Integer> year;
    public static volatile SingularAttribute<Car, String> licence;
    public static volatile SingularAttribute<Car, Spot> spot;
    public static volatile SingularAttribute<Car,Boolean> active;
    public static volatile SingularAttribute<Car, BigDecimal> price;

}
